docs/2-tutorials/2-develop/6-push-notifications.html.md

```diff
@@ -2,7 +2,164 @@
 title: Push Notifications
 url: develop/push-notifications
 layout: subpage
-write: false
 ---
 
-Push Notifications content here
+You can use the **PhoneGap Developer App** paired with the **PhoneGap CLI** to immediately preview your app on a device without installing platform SDKs, registering devices, or compiling code. The PhoneGap CLI starts a small web server to host your project and returns the server address for you to pair with from the PhoneGap Developer App running on your mobile device.
+
+As of **PhoneGap Developer App** release 1.6.2 and **PhoneGap CLI** release 6.2.0 you can now test push notification functionality without needing to setup a GCM project for Android or an Apple push certification for iOS.
+
+<div class="alert--warning">Double check to ensure you're running your device and computer on the same network before continuing. </div>
+
+### Create Default PhoneGap Push Project
+
+1. Enter the following command from your terminal:
+
+		$ phonegap create myApp --template phonegap-template-push
+
+   This will create a folder named myApp in the current path location with a default project name of Hello World and id of com.phonegap.helloworld.
+
+   You can also specify a name and identifier to ensure the project is unique but still contains the default Hello World code project by specifying them as qualified parameters as shown below:
+
+		$ phonegap create myApp --id "org.myapp.sample" --name "appSample" --template phonegap-template-push
+
+2. Verify that you see the following output in your console after you run the command:
+
+		Creating a new cordova project.
```

Comment:
You may want to mention you see both of these lines in your console after you run the command:

     Creating a new cordova project.

     Retrieving phonegap-template-push using npm...